neon_iris/web_client.py

```diff
@@ -118,14 +118,14 @@ def update_profile(self, stt_lang: str, tts_lang: str, tts_lang_2: str,
     def on_user_input(self, utterance: str,
                       chat_history: List[Tuple[str, str]],
                       audio_input: str,
-                      client_session: str) -> (List[Tuple[str, str]], str, str, None, str):
+                      client_session: str):
```

Comment:
Was this annotation inaccurate?

Reply:
Yes - it was making MyPy go crazy. I can go back and pull the accurate one and add it, if you'd like.